src/wp-includes/class-wp-script-modules.php

```diff
@@ -151,6 +151,18 @@ public function dequeue( string $id ) {
 		unset( $this->enqueued_before_registered[ $id ] );
 	}
 
+	/**
+	 * Removes a registered script module.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $id The identifier of the script module.
+	 */
+	public function deregister( string $id ) {
```

Comment:
Is it "unregister" or "deregister"? It seems on Core, we use "unregister" everywhere except for WP_Scripts and Styles. It seems a bit awkward. 

I understand the reasoning about modules being scripts to keep them consistent but wanted to mention.

Reply:
Right, that's unfortunate. It's probably better to keep it awkward but similar to scripts. I don't have strong opinion here though 😅

Reply:
It's even more interesting:

https://github.com/WordPress/wordpress-develop/blob/72bc9c0b44a92ad1b657029b648055c0c0a1f601/src/wp-includes/functions.wp-scripts.php#L278

but:

https://github.com/WordPress/wordpress-develop/blob/72bc9c0b44a92ad1b657029b648055c0c0a1f601/src/wp-includes/functions.wp-scripts.php#L334